src/Functions/array/arrayAggregation.cpp

```diff
@@ -146,8 +146,89 @@ struct ArrayAggregateImpl
 
         const ColVecType * column = checkAndGetColumn<ColVecType>(&*mapped);
 
+        /// Constant case.
         if (!column)
-            return false;
+        {
```

Comment:
@kitaisreal In which situation will execute the constant branch?
Is there any examples that can trigger this const branch?

Reply:
Actually yes, I merged this pull request in separate branch to remove condition and check if tests will work, actually some of tests were broken, so I put it back.